tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -2,9 +2,70 @@
 # SPDX-License-Identifier: Apache-2.0
 """Tests for vhost-user-block device."""
 
+import os
+from subprocess import check_output
+
+import pytest
+
+import host_tools.drive as drive_tools
 from framework import utils
+from framework.defs import LOCAL_BUILD_PATH
 from framework.utils_vhost_user_backend import spawn_vhost_user_backend
 
+MB = 1024 * 1024
+
+
+@pytest.fixture
+def partuuid_and_disk_path(rootfs_ubuntu_22):
```

Comment:
can we re-use the existing one in https://github.com/firecracker-microvm/firecracker/blob/main/tests/integration_tests/functional/test_drives.py#L17 and then just move it where you need it?

Reply:
I guess I can, but at the same time I feel like it might be better to update the `uvm_with_partuuid` to just create a file with partuuid (like what `partuuid_and_disk_path` does). Then it would be simple to reuse it. What do you think?

Reply:
Maybe we can have both fixtures and they will just call 1 function that accepts parameter if file needs to be in tmpfs or on the host.

Reply:
Extracting that bit so that it is a separate function? Yeah that sounds good.

Reply:
Already done